$NetBSD: patch-ea,v 1.1.2.2 2010/08/16 12:26:22 tron Exp $

* Fix for possible cross-site scripting (CVE-2010-0541) from r29002 in
  Ruby's repository.

--- lib/webrick/httpresponse.rb.orig	2008-06-06 08:05:24.000000000 +0000
+++ lib/webrick/httpresponse.rb
@@ -209,7 +209,7 @@ module WEBrick
         @keep_alive = false
         self.status = HTTPStatus::RC_INTERNAL_SERVER_ERROR
       end
-      @header['content-type'] = "text/html"
+      @header['content-type'] = "text/html; charset=ISO-8859-1"
 
       if respond_to?(:create_error_page)
         create_error_page()
